k9mail/src/test/java/com/fsck/k9/helper/ReplyToParserTest.java

```diff
@@ -0,0 +1,56 @@
+package com.fsck.k9.helper;
+
+import com.fsck.k9.mail.Address;
+import com.fsck.k9.mail.Message;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+import static org.junit.Assert.assertSame;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@RunWith(RobolectricTestRunner.class)
+@Config(manifest = Config.NONE, sdk = 21)
+public class ReplyToParserTest {
+    private Message message = mock(Message.class);
+    private Address[] replyToAddresses = new Address[]{new Address("replyTo@example.com")};
+    private Address[] listPostAddresses = new Address[]{new Address("listPost@example.com")};
+    private Address[] fromAddresses = new Address[]{new Address("from@example.com")};
+
+    @Before
+    public void setUp() throws Exception {
+        when(message.getReplyTo()).thenReturn(new Address[]{});
```

Comment:
I prefer `new Address[0]` because I think it's easier to read than `new Address[]{}`.